src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -894,6 +894,7 @@ private static bool AddHidingAbstractDiagnostic(Symbol hidingMember, Location hi
             {
                 case Accessibility.Internal:
                 case Accessibility.Private:
+                case Accessibility.ProtectedAndInternal:
```

Comment:
It is not obvious why this shouldn't be handled the same way as other protected.